tests/unit/test_utils.py

```diff
@@ -409,3 +410,25 @@ def test_can_escape_recursive_shapes(self):
         }
         self.walker.walk(self.get_shape_model('Recursive'), self.visitor)
         self.assert_visited_shapes(['Recursive'])
+
+
+@pytest.fixture(scope='module')
+def argument_model():
+    return botocore.model.Shape('', {'type': ''})
```

Comment:
For the `Shape` instantiation, let's use realistic values. I'm fine giving it the name `argument` and its type `string`.